### ISSUE TITLE
PDF accessibility validation

### ISSUE THREAD

**Issue by ecureuill** [solex2006/SELIProject#149]
- [ ]  Accessibility Syllabus Validation :pencil2: 
  - https://accessibility.umn.edu/documents/creating-accessible-pdfs
  - https://www.w3.org/TR/WCAG20-TECHS/pdf
  - https://helpx.adobe.com/acrobat/using/create-verify-pdf-accessibility.html

**Comment by CAndresH** [comment]
Camila I have a question: Where on the platform should these radio buttons go?
![image](https://user-images.githubusercontent.com/50594090/87863706-790aa300-c923-11ea-9683-53e0db372e42.png)

in the part of configure accessibility?, after upload an image?,
![image](https://user-images.githubusercontent.com/50594090/87863714-9b9cbc00-c923-11ea-8fe2-ea6e4f972635.png)
after uploading the pdf should appear this menu with the option configure accessibility and configure accessibility later, right?
![image](https://user-images.githubusercontent.com/50594090/87863728-e9192900-c923-11ea-81cb-638c24dcca9a.png)



 the same for the other options: Navigation,text content, form and structures, should they go the same way in a menu like this?
![image](https://user-images.githubusercontent.com/50594090/87863755-5cbb3600-c924-11ea-8844-334b7e861a66.png)

**Comment by CAndresH** [comment]
Another question is how do these new accessibility parameters affect the reports?

for example now we have parameters like hasTitle, hasNumbering, hasLanguage, screenReader and others, and I don't know what type of disability they configure for example: screenReader configures visual impairment?

**Comment by CAndresH** [comment]
This type of accessibility should be created only for the part of the syllable or when a teacher uploads a pdf in a course as well?

**Comment by ecureuill** [comment]
Hi Andres! 
Exactly, this radios will be part of the Accessibilty Configuration after teacher upload an PDF on Program step and also after upload syllabus. 
And each item that is a Tab in this issue, should be a tab/navigation in the Accessibility Configuration form.

**About the report.**
For each accessibility option, check for the "well configured" rule, they have an related issue
![image](https://user-images.githubusercontent.com/993369/87997523-b742c680-cacb-11ea-91bb-ee7ddb04bd1b.png)
If you open the related issue, you can check the labels to see which disabilities it affects
![image](https://user-images.githubusercontent.com/993369/87997536-bf9b0180-cacb-11ea-8ae4-f153605401ed.png)
In above example, thus, if teacher selected to make the course accessible to *cognitive* or *visual* then this item should be calculated in the report

**Comment by CAndresH** [comment]
Hello @rebeccamello @caroltaus, @MAmelia  , I understand this must go under the option to upload the pdf ?, but I can not do this requirement if you do not upload the information that should go in this part, remember that Maria Amelia in the meetings some time ago said that she was going to upload the information that we must place in the "More datails" and "Help me decide" options, but we still don't have that information

**Comment by caroltaus** [comment]
can you please provide this information? @flaviolima  thank you!

**Comment by CAndresH** [comment]
@rebeccamello  @caroltaus @flaviolima, Hello team, where is the information that should be placed in this part?,

**Comment by flaviolima** [comment]
Hi @CAndresH I'm providing this information and will be available in the next days. Thanks.

**Comment by caroltaus** [comment]
Info was provided by @flaviolima at https://github.com/solex2006/SELIProject/issues/46
